changelogs/fragments/gem_module_add_bindir_option.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - gem - add ``bindir`` option to specify an installation path for executables such as ``/home/user/bin`` or ``/home/user/.local/bin`` (https://github.com/ansible-collections/community.general/pull/2837).
+  - gem - add ``norc`` option to avoid loading any .gemrc file (https://github.com/ansible-collections/community.general/pull/2837).
```

Comment:
```suggestion
  - gem - add ``norc`` option to avoid loading any ``.gemrc`` file (https://github.com/ansible-collections/community.general/pull/2837).
```